js/main.js

```diff
@@ -236,3 +270,29 @@ document.addEventListener('DOMContentLoaded', function (event) {
     $yourReviewsList.appendChild(reviewResult);
   }
 });
+
+$yourReviewsList.addEventListener('click', function (event) {
+  if (event.target && event.target.matches('BUTTON')) {
+    var $liClosest = event.target.closest('li');
+    var $reviewsId = $liClosest.getAttribute('data-review-id');
+
+    $reviewsId = JSON.parse($reviewsId);
```

Comment:
You should use `parseInt`, not `JSON.parse`

Reply:
<img width="435" alt="Screen Shot 2022-08-04 at 2 58 38 PM" src="https://user-images.githubusercontent.com/97363006/182960364-6c362201-188e-4c16-bedc-c4d7c8039568.png">